website/docs/guides/1.14.0-upgrade-guide.html.markdown

```diff
@@ -0,0 +1,35 @@
+---
+layout: "mongodbatlas"
+page_title: "MongoDB Atlas Provider 1.14.0: Upgrade and Information Guide"
+sidebar_current: "docs-mongodbatlas-guides-1140-upgrade-guide"
+description: |-
+MongoDB Atlas Provider 1.14.0: Upgrade and Information Guide
+---
+
+# MongoDB Atlas Provider 1.14.0: Upgrade and Information Guide
+
+The Terraform MongoDB Atlas Provider version 1.14.0 has a number of new and exciting features.
+
+**New Resources, Data Sources, and Features:**
+- There are new resources related to [Atlas Stream Processing](https://www.mongodb.com/docs/atlas/atlas-sp/overview/#atlas-stream-processing-overview). 
+    - `mongodbatlas_stream_instance` [resource](https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/resources/stream_instance) and data sources ([singular](https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/data-sources/stream_instance) and [paginated](https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/data-sources/stream_instances)) enable managing [Atlas Stream Instances](https://www.mongodb.com/docs/atlas/reference/api-resources-spec/#tag/Streams/operation/createStreamInstance).
+    - `mongodbatlas_stream_connection` [resource](https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/resources/stream_connection) and data sources ([singular](https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/data-sources/stream_connection) and [paginated](https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/data-sources/stream_connections)) enable managing [Atlas Stream Instance Connections](https://www.mongodb.com/docs/atlas/reference/api-resources-spec/#tag/Streams/operation/createStreamConnection). 
+
+**Breaking Changes:**
```

Comment:
Can you link each of these breaking changes to specific PRs so users can learn more as needed?

Reply:
will do